src/Storage/StorageClient.php

```diff
@@ -217,4 +223,53 @@ public function createBucket($name, array $options = [])
         $response = $this->connection->insertBucket($options + ['name' => $name, 'project' => $this->projectId]);
         return new Bucket($this->connection, $name, $response);
     }
+
+    /**
+     * Register a StreamWrapper for reading and writing to Google Storage
+     *
+     * @param string $protocol The name of the protocol to use. Defaults to
+     *        'gs'.
+     * @throws \RuntimeException
+     */
+    public static function registerStreamWrapper(string $protocol = null)
+    {
+        $protocol = $protocol ?: 'gs';
+        if (!in_array($protocol, stream_get_wrappers())) {
+            if (!stream_wrapper_register($protocol, StreamWrapper::class)) {
+                throw new RuntimeException("Failed to register '$protocol://' protocol");
+            }
+        }
+    }
+
+    /**
+     * Unregisters the SteamWrapper
+     *
+     * @param string $protocol The name of the protocol to unregister. Defaults
+     *        to 'gs'.
+     */
+    public static function unregisterStreamWrapper(string $protocol = null)
+    {
+        stream_wrapper_unregister($protocol ?: 'gs');
+    }
+
+    /**
+     * Get the default client to use for global interactions with storage.
+     *
+     * @return StorageClient
+     */
+    public static function getDefaultClient()
+    {
+        return self::$defaultClient;
```

Comment:
Perhaps `registerAsStreamWrapper` to indicate that it's using this instance of the client.

Is it possible that someone may instantiate a second `StorageClient` and try to re-register? Perhaps a raise a warning and/or return false? Or even just swap out the default client and let it work. We should probably be explicit about what the user should expect in that (edge) case.